website/members/fixtures/members.json

```diff
@@ -216,7 +216,8 @@
             "type": "member",
             "user": 1,
             "since": "1980-01-01",
```

Comment:
It's kind of weird to still have type=member+until=None in the fixtures, as that should be impossible after the migration.